tests/methods/avas-7/input.dat

```diff
@@ -0,0 +1,41 @@
+# Test the pi orbital selection of AOSubspace
+# Same as AVAS-1 test with rotated xyz (plane normal: [1,1,1])
+
+import forte
+
+refscf = -113.87663391962522
+reffci = -113.911667467206257
+
+molecule {
+0 1
+C        0.346146295209737    0.126698337466632   -0.472844632676369
+O       -0.346053592352928   -0.126664405871036    0.472717998223964
+H        1.227335215970831   -0.489581944167415   -0.737753271803415
+H        0.143281782803090    0.991262584630455   -1.134544367433545
+noreorient
+symmetry c1
+}
+
+set {
+  basis cc-pvdz
+  reference rhf
+  scf_type pk
+  e_convergence 12
+}
+
+set forte {
+  subspace ["C(2p)","O(2P)"]  # must include all p orbitals
+  subspace_pi_planes [["C","O","H"]]  # a list of planes
```

Comment:
Here would be a place to specify what plane the orbitals are on. Similar to before like on the xz plane

Reply:
The plane function is 0 = (x - Cx) + (y - Cy) + (z - Cz) = (x - Ox) + (y - Oy) + (z - Oz) = (x - H1x) + (y - H1y) + (z - H1z) = (x - H2x) + (y - H2y) + (z - H2z) because the plane normal is (1, 1, 1).

Reply:
I think the comment suggests that it's not obvious what is going on here. A simple comment like "Selected the molecular plane defined by all the C, O, and H atoms" would do.

Reply:
I explained more in the text.